tests-ng/plugins/module.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from .shell import ShellRunner
+
+class Module:
+    def __init__(self, shell: ShellRunner):
+        self._shell = shell
+
+    def is_module_loaded(self, module: str) -> bool:
+        result = self._shell(f"lsmod | grep {module}", capture_output=True, ignore_exit_code=True)
```

Comment:
`grep '^{module} ' /proc/modules` or something similar in python would make 100% sure we match the correct module.